Build Method Should Be Updated in the Installation Docs

In the installation docs:

https://openllb.github.io/hlb/intro/quickstart/#installation

A link to a Releases page is provided, but the Darwin binaries fail to run on MacOS Big Sur, due to not being signed by a recognized developer. As a work-around for this, I tried building the cli from source. I figure that this is also a reason why we provide access to this tool through newt.

On the installation page, it is recommended to use the following process to build the `hlb` cli tool:

`go get -u github.com/openllb/hlb/cmd/hlb`

This process fails with errors:

```
❯ go get -u github.com/openllb/hlb/cmd/hlb 
# github.com/openllb/hlb/pkg/imageutil
src/github.com/openllb/hlb/pkg/imageutil/buffered_resolver.go:71:23: cannot use bir (type *BufferedImageResolver) as type "github.com/containerd/containerd/content".Ingester in argument to "github.com/containerd/containerd/remotes".FetchHandler:
	*BufferedImageResolver does not implement "github.com/containerd/containerd/content".Ingester (wrong type for Writer method)
		have Writer(context.Context, ..."github.com/moby/buildkit/vendor/github.com/containerd/containerd/content".WriterOpt) ("github.com/moby/buildkit/vendor/github.com/containerd/containerd/content".Writer, error)
		want Writer(context.Context, ..."github.com/containerd/containerd/content".WriterOpt) ("github.com/containerd/containerd/content".Writer, error)
src/github.com/openllb/hlb/pkg/imageutil/buffered_resolver.go:72:25: cannot use bir (type *BufferedImageResolver) as type "github.com/containerd/containerd/content".Provider in argument to images.ChildrenHandler:
	*BufferedImageResolver does not implement "github.com/containerd/containerd/content".Provider (wrong type for ReaderAt method)
		have ReaderAt(context.Context, "github.com/moby/buildkit/vendor/github.com/opencontainers/image-spec/specs-go/v1".Descriptor) ("github.com/moby/buildkit/vendor/github.com/containerd/containerd/content".ReaderAt, error)
		want ReaderAt(context.Context, "github.com/containerd/containerd/vendor/github.com/opencontainers/image-spec/specs-go/v1".Descriptor) ("github.com/containerd/containerd/content".ReaderAt, error)
src/github.com/openllb/hlb/pkg/imageutil/buffered_resolver.go:76:23: cannot use desc (type "github.com/opencontainers/image-spec/specs-go/v1".Descriptor) as type "github.com/containerd/containerd/vendor/github.com/opencontainers/image-spec/specs-go/v1".Descriptor in argument to images.Dispatch
src/github.com/openllb/hlb/pkg/imageutil/buffered_resolver.go:85:22: cannot use *opt.Platform (type "github.com/moby/buildkit/vendor/github.com/opencontainers/image-spec/specs-go/v1".Platform) as type "github.com/containerd/containerd/vendor/github.com/opencontainers/image-spec/specs-go/v1".Platform in argument to platforms.Only
src/github.com/openllb/hlb/pkg/imageutil/buffered_resolver.go:88:30: cannot use bir (type *BufferedImageResolver) as type "github.com/containerd/containerd/content".Provider in argument to images.Config:
	*BufferedImageResolver does not implement "github.com/containerd/containerd/content".Provider (wrong type for ReaderAt method)
		have ReaderAt(context.Context, "github.com/moby/buildkit/vendor/github.com/opencontainers/image-spec/specs-go/v1".Descriptor) ("github.com/moby/buildkit/vendor/github.com/containerd/containerd/content".ReaderAt, error)
		want ReaderAt(context.Context, "github.com/containerd/containerd/vendor/github.com/opencontainers/image-spec/specs-go/v1".Descriptor) ("github.com/containerd/containerd/content".ReaderAt, error)
src/github.com/openllb/hlb/pkg/imageutil/buffered_resolver.go:88:30: cannot use desc (type "github.com/opencontainers/image-spec/specs-go/v1".Descriptor) as type "github.com/containerd/containerd/vendor/github.com/opencontainers/image-spec/specs-go/v1".Descriptor in argument to images.Config
src/github.com/openllb/hlb/pkg/imageutil/buffered_resolver.go:93:29: cannot use bir (type *BufferedImageResolver) as type "github.com/containerd/containerd/content".Provider in argument to "github.com/containerd/containerd/content".ReadBlob:
	*BufferedImageResolver does not implement "github.com/containerd/containerd/content".Provider (wrong type for ReaderAt method)
		have ReaderAt(context.Context, "github.com/moby/buildkit/vendor/github.com/opencontainers/image-spec/specs-go/v1".Descriptor) ("github.com/moby/buildkit/vendor/github.com/containerd/containerd/content".ReaderAt, error)
		want ReaderAt(context.Context, "github.com/containerd/containerd/vendor/github.com/opencontainers/image-spec/specs-go/v1".Descriptor) ("github.com/containerd/containerd/content".ReaderAt, error)
src/github.com/openllb/hlb/pkg/imageutil/buffered_resolver.go:102:27: cannot use bir.defaultPlatform (type "github.com/opencontainers/image-spec/specs-go/v1".Platform) as type "github.com/containerd/containerd/vendor/github.com/opencontainers/image-spec/specs-go/v1".Platform in argument to platforms.Only
src/github.com/openllb/hlb/pkg/imageutil/buffered_resolver.go:113:47: cannot use r.Digest() (type "github.com/containerd/containerd/vendor/github.com/opencontainers/go-digest".Digest) as type "github.com/opencontainers/go-digest".Digest in argument to bir.DigestDescriptor
src/github.com/openllb/hlb/pkg/imageutil/buffered_resolver.go:118:16: cannot assign "github.com/containerd/containerd/vendor/github.com/opencontainers/image-spec/specs-go/v1".Descriptor to desc (type "github.com/opencontainers/image-spec/specs-go/v1".Descriptor) in multiple assignment
src/github.com/openllb/hlb/pkg/imageutil/buffered_resolver.go:118:16: too many errors
# github.com/openllb/hlb/pkg/sockproxy
src/github.com/openllb/hlb/pkg/sockproxy/sockprovider.go:64:9: cannot use &socketProvider literal (type *socketProvider) as type session.Attachable in return argument:
	*socketProvider does not implement session.Attachable (wrong type for Register method)
		have Register(*"google.golang.org/grpc".Server)
		want Register(*"github.com/moby/buildkit/vendor/google.golang.org/grpc".Server)
src/github.com/openllb/hlb/pkg/sockproxy/sockprovider.go:78:30: cannot use server (type *"google.golang.org/grpc".Server) as type *"github.com/moby/buildkit/vendor/google.golang.org/grpc".Server in argument to sshforward.RegisterSSHServer
```

The README in the project provides another method of building the cli, which works and the binary shows up in `$GOPATH/bin`:

```
git clone https://github.com/openllb/hlb.git
cd hlb
go install ./cmd/hlb
```

The MkDocs site should be updated to match the README.

--- a/docs/intro/quickstart.md
+++ b/docs/intro/quickstart.md
@@ -6,7 +6,9 @@ If you're on a MacOS or Linux (`linux-amd64`), head on over to [Releases](https:
 
 Otherwise, you can compile HLB yourself using [go](https://golang.org/dl/):
 ```sh
-go get -u github.com/openllb/hlb/cmd/hlb
+git clone https://github.com/openllb/hlb.git
+cd hlb
+go install ./cmd/hlb
 ```
 
 You'll also need to run `buildkitd` somewhere you can connect to. The easiest way if you have [Docker](https://www.docker.com/get-started), is to run a local buildkit container:

